File: 389-find-the-difference/389-find-the-difference.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
        int a=0,b=0;
        for(char c:s){
            a+=c;
        }
        for(char d:t){
            b+=d;
        }
        return b-a;
    }
};

File: 389-find-the-difference/389-find-the-difference.java
class Solution {
    public char findTheDifference(String s, String t) {
        char res=t.charAt(s.length());
        for(int i=0;i<s.length();i++){
            res^=s.charAt(i);
            res^=t.charAt(i);
        }
        return res;
        
        
    }
}